ui/src/components/mobileservices/BindingPanel.js

```diff
@@ -77,37 +78,27 @@ export class BindingPanel extends Component {
   }
 
   show() {
+    this.reinitState();
     this.stepChanged(0);
     this.open();
   }
 
-  hasUPSAndroidAnnotation() {
-    return this.hasUPSPlatformAnnotation('android');
+  hasUPSIOSBoundService() {
+    return this.hasUPSIOSBoundServiceForPlatform('ios');
   }
 
-  hasUPSIOSAnnotation() {
-    return this.hasUPSPlatformAnnotation('ios');
+  hasUPSAndroidBoundService() {
+    return this.hasUPSIOSBoundServiceForPlatform('android');
   }
 
-  hasUPSPlatformAnnotation(platform) {
-    // there won't be any variant annotations if there is no binding yet
-    if (!this.props.service.isBound()) {
-      return false;
-    }
-
-    const configExtItems = this.props.service.getConfigurationExtAsJSON();
-    if (!configExtItems || !configExtItems.length) {
-      return false;
-    }
-
-    for (const configExtItem of configExtItems) {
-      for (const variantInfo of configExtItem) {
-        if (variantInfo.type === platform) {
-          return true;
-        }
-      }
-    }
-    return false;
+  hasUPSIOSBoundServiceForPlatform(platform) {
```

Comment:
should this be `hasUPSBoundServiceForPlatform` instead of `hasUPSIOSBoundServiceForPlatform`?

Reply:
👍